frontend/components/forms/ConfirmInviteForm/helpers.jsx

```diff
@@ -0,0 +1,38 @@
+import { size } from "lodash";
+import validateEquality from "components/forms/validators/validate_equality";
```

Comment:
This just calls lodash's `isEqual`. Using here for consistency but perhaps we can retire this function in favor of calling the lodash method directly.